app/SpotifyPlayActivity.php

```diff
@@ -4,14 +4,20 @@
 
 use Illuminate\Database\Eloquent\Model;
 
-class SpotifyPlayActivity extends Model {
+class SpotifyPlayActivity extends Model
+{
 
-    protected $fillable = [
-        'user_id', 'track_id', 'timestamp_start', 'progress_ms', 'context', 'context_uri', 'device_id'
-    ];
+    protected $fillable = ['user_id', 'track_id', 'timestamp_start', 'progress_ms', 'context', 'context_uri', 'device_id'];
+    protected $dates = ['timestamp_start'];
```

Comment:
Die `=` sind nicht ausgerichtet.

Reply:
I'm not sure if I would like to do this.